### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by nagasai-iitr**
The requirements code corresponding to python 2.4

### CODE REVIEW

**Comment by nagasai-iitr** [comment]
Please look into this

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Make sure you have installed the following packages from the Ubuntu repository:
     sudo apt-get update
     sudo apt-get install git git-annex cmake make g++ libgsl-dev libopenscenegraph-dev cmake-qt-gui zeroc-ice-all-dev freeglut3-dev libboost-system-dev libboost-thread-dev qt5-default libqt5xmlpatterns5-dev libqt5opengl5-dev libxt-dev libboost-test-dev libboost-filesystem-dev  libccd-dev zeroc-ice-all-runtime
     sudo apt-get install python3-pip python3-setuptools python3-pyparsing=2.2.0+dfsg1-2 python3-numpy python3-libxml2 python3-xmltodict python3-zeroc-ice
-    sudo pip3 install networkx pyside2 argcomplete termcolor cogapp
+    sudo pip install networkx pyside2 argcomplete termcolor cogapp
```
Comment (orensbruli) [inline README.md]: why this change?

Comment (nagasai-iitr) [reply README.md]: we here are mainly using python 2 primarily currently so is the reason for change
During my installation I have been stuck because of this too

Comment (orensbruli) [reply README.md]: Who is using mainly python2?
We have moved all the Robocomp core apps to python3, the apt-get command above of pip is installing python3 and related libs... 
Some of the components in robocomp-robolab repository are in python2 but it's a task for each developer (or contributor) to migrate it to python3.
We are going back to python2.

Comment (nagasai-iitr) [reply README.md]: The thing is that while I was exploring Key Board controller I had an issue regarding the PySide2 
Which got solved with the above edit

Comment (nagasai-iitr) [reply README.md]: Until the all the components are capable of python3 ,I thought the requirements should be regarding the needs of the current components

Comment (orensbruli) [reply README.md]: Thank for your opinion.
Python2 will not be supported anymore (https://www.python.org/doc/sunset-python-2/).
None of us knows all the components and it's mostly impossible to move all to python3. 
All the main tools are already migrated.
If you want to make your contribution migrating to python3 whatever component is not working in python2 it will be welcomed.
